crates/turborepo-lib/src/cli/error.rs

```diff
@@ -53,3 +55,51 @@ pub enum Error {
     #[diagnostic(transparent)]
     Watch(#[from] watch::Error),
 }
+
+const MAX_CHARS_PER_TASK_LINE: usize = 100;
+
+pub async fn print_potential_tasks(
+    base: CommandBase,
+    telemetry: CommandEventBuilder,
+) -> Result<(), Error> {
+    let signal = get_signal()?;
+    let handler = SignalHandler::new(signal);
+    let ui = base.ui;
+
+    let run_builder = RunBuilder::new(base)?;
+    let run = run_builder.build(&handler, telemetry).await?;
+    let potential_tasks = run.get_potential_tasks()?;
+
+    println!("No tasks provided, here are some potential ones to run\n",);
+
+    for (task, packages) in potential_tasks
+        .into_iter()
+        .sorted_by(|(_, a), (_, b)| b.len().cmp(&a.len()))
+    {
+        let task = color!(ui, BOLD, "{}", task);
```

Comment:
[praise](https://conventionalcomments.org/): the extra attention you took to contrast the headings from the items makes it REALLY nice on the eyes.  it's really appreciated:

![Screenshot_20240715_161511](https://github.com/user-attachments/assets/eea4d2f8-c1aa-46d3-b2c5-e524aec81674)